src/jvm/test/resources/expected_logs/custom_threads_deque_trace.txt

```diff
@@ -0,0 +1,78 @@
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| incorrectConcurrentLinkedDeque(): IllegalStateException #1                                                                                     |                                                                                                                                                             |          |
+|   ConcurrentLinkedDeque#1.addLast(1) at CustomThreadsRepresentationTest.incorrectConcurrentLinkedDeque(CustomThreadsRepresentationTest.kt:112) |                                                                                                                                                             |          |
+|   THREAD_START(2)                                                                                                                              |                                                                                                                                                             |          |
+|   THREAD_START(3)                                                                                                                              |                                                                                                                                                             |          |
+|   switch                                                                                                                                       |                                                                                                                                                             |          |
+|                                                                                                                                                | run()                                                                                                                                                       |          |
+|                                                                                                                                                |   switch                                                                                                                                                    |          |
+|   switch (reason: waiting for thread to join)                                                                                                  |                                                                                                                                                             |          |
```

Comment:
let's mention which thread the execution is calling `join` on

Reply:
Done